PhysicsTools/NanoAOD/python/taus_cff.py

```diff
@@ -1,43 +1,38 @@
 import FWCore.ParameterSet.Config as cms
 from PhysicsTools.NanoAOD.common_cff import *
 from PhysicsTools.JetMCAlgos.TauGenJets_cfi import tauGenJets
-from PhysicsTools.JetMCAlgos.TauGenJetsDecayModeSelectorAllHadrons_cfi import tauGenJetsSelectorAllHadrons 
-
-##################### Updated tau collection with MVA-based tau-Ids rerun #######
-# Used only in some eras
-from PhysicsTools.NanoAOD.taus_updatedMVAIds_cff import *
+from PhysicsTools.JetMCAlgos.TauGenJetsDecayModeSelectorAllHadrons_cfi import tauGenJetsSelectorAllHadrons
 
 ##################### User floats producers, selectors ##########################
 
-
 finalTaus = cms.EDFilter("PATTauRefSelector",
-    src = cms.InputTag("slimmedTausUpdated"),
-    cut = cms.string("pt > 18 && tauID('decayModeFindingNewDMs') && (tauID('byLooseCombinedIsolationDeltaBetaCorr3Hits') || tauID('byVLooseIsolationMVArun2v1DBoldDMwLT2015') || tauID('byVLooseIsolationMVArun2v1DBnewDMwLT') || tauID('byVLooseIsolationMVArun2v1DBdR03oldDMwLT') || tauID('byVVLooseIsolationMVArun2v1DBoldDMwLT') || tauID('byVVLooseIsolationMVArun2v1DBoldDMwLT2017v2') || tauID('byVVLooseIsolationMVArun2v1DBnewDMwLT2017v2') || tauID('byVVLooseIsolationMVArun2v1DBdR03oldDMwLT2017v2'))")
+    src = cms.InputTag("slimmedTausNewID"),
```

Comment:
Propose to keep previously used name of updated tau collection, i.e. `slimmedTausUpdated` (it applies also to other places in `PhysicsTools/NanoAOD/python/nano_cff.py`)